src/app/index.tsx

```diff
@@ -70,10 +73,24 @@ export default (options: Options) => {
     reducer,
   });
 
+  const language = (typeof window === 'undefined') ? 'en-us' : window.navigator.language;
+  addLocaleData([...en, ...cs]);
+
+  const messages = new Map()
+  messages.set('en', { 'i18n:404': 'page not found' });
+  messages.set('cs', { 'i18n:404': 'požadovaná stránka neexistuje'});
+
+  function getMessages(language: string) {
+      const lang = language.split('-')[0]
+      return messages.get(lang) || messages.get('en')
```

Comment:
you're defaulting to english in two places (here and on 76) we should consolidate this